feat: yivi conditional disclosure

marnixdessing opened the pull request:
- Use scopes returned by IdP to check claims present
- Make authentication logic very explicit

marnixdessing committed:
feat: prepare for conditional disclosure using scopes and claims

WietekeNijmegen commented on src/app/auth/AuthRequestHandler.ts:
```diff
@@ -34,31 +36,37 @@ export class AuthRequestHandler {
+      const claims = tokens.claims();
+      const scope = tokens.scope;
+      if (!claims || !scope) {
         return Response.redirect('/login');
```
Gaat dit niet mis? Je wil toch dat het op regel 72 gebeurt

joostvanderborg committed:
chore: A bit of refactoring

Moving the claims & scope out of the handler, since they're not really
used there.

marnixdessing committed:
Merge pull request #1077 from GemeenteNijmegen/feat/yivi-conditional-disclosure-suggestions

chore: A bit of refactoring

marnixdessing committed:
chore: add additional parameters and environment variables

marnixdessing committed:
fix: descriptions and missing eherkenning oidc scope

marnixdessing committed:
chore: update tests

marnixdessing committed:
Merge branch 'acceptance' into feat/yivi-conditional-disclosure
